packages/app/background-jobs-common/src/background-job-processor/utils.ts

```diff
@@ -16,3 +20,27 @@ export const isStalledJobError = (error: Error): boolean =>
 export const sanitizeRedisConfig = (config: RedisConfig): RedisConfig => {
```

Comment:
wonder if we can improve naming a bit, "sanitization" usually refers to removing unsafe values, and in this case this is a slightly different kind of transformation.

Reply:
Do you have any particular name in mind? I don't have a strong opinion but for me `sanitize` for me sounds valid as we are removing from the config those fields that are not supported by BullMQ (will produce an error on init)

Reply:
maybe, if it's not confusing for you, then it's good enough, let's keep it

Reply:
I am more than open to change it, not saying it is perfect but nothing better comes to my mind (what is normal for me, I am super bad with names 😓) so please if you have a better one in mind share it and I will update it :D  

Also, note that this is a library private method, it is not exposed